circle.yml

```diff
@@ -15,7 +15,12 @@ machine:
 dependencies:
     pre:
         - echo y | android update sdk --no-ui --all --filter "tools"
-        - echo y | android update sdk --no-ui --all --filter "build-tools-25.0.0"
+        - echo y | android update sdk --no-ui --all --filter "build-tools-25.2.3"
+        - echo y | android update sdk --no-ui --all --filter "extra-android-m2repository"
+        - echo y | android update sdk --no-ui --all --filter "extra-android-support"
+        - echo y | android update sdk --no-ui --all --filter "extra-google-google_play_services"
```

Comment:
HM I think google-play-services might be removed. For other I think we needed as build-tools include sdkmanager which manages our licenses and other are added as per reference to our solution. What should I do? I don't think adding this will cause any issue.